src/scheduler/workers/processListsBeforeAndDuringStart/main.ts

```diff
@@ -145,8 +136,8 @@ export async function processList(list: List, listItemsForList: ListItemWithHist
     return;
   }
   // get the most recent audit record to determine if the email has already been sent for the respective milestones
-  const audit = await getLatestReminderAuditEvent(annualReviewRef, "list");
-  const listItemAudit = await getLatestReminderAuditEvent(annualReviewRef, "listItem");
+  const audit = await getLatestReminderEvent(async () => await findReminderAuditEvents(annualReviewRef));
+  const listItemAudit = await getLatestReminderEvent(async () => await findAllReminderEvents(annualReviewRef));
```

Comment:
Maybe it's too late in the day.. but I wonder if this logic is correct for l140 / provider emails.. This just gets the latest event for _all_ list items (with the same annual review ref). 

If some emails failed, it means that the failed emails are never retried because it succeeded for someone else.

Reply:
Hmmm, I guess the best solution would be to check if the list id's match and the timeframe of the event matches when the email was sent.

This could be applied to the audit as well. 

I'll see what I can do.